contrib/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM readthedocs/build:latest
+
+ARG username
+ARG uid=$uid
+ARG gid=$uid
+
+ENV UID ${uid}
+ENV GID ${gid}
+
+USER root
+RUN groupadd --gid ${GID} docscustom
```

Comment:
@humitos Why did we remove the existing groupadd for this? If we still need to add a group, might as well not use `docscustom` for a string.

Reply:
i changed this to groupmod, which seems like the obvious answer if we're just modifying the user/group